Jenkinsfile

```diff
@@ -1,14 +1,13 @@
 @Library('shared-libraries') _
 
-def runtests(String mlVersionType, String mlVersion, String javaVersion){
-  copyRPM mlVersionType,mlVersion
-  setUpML '$WORKSPACE/xdmp/src/Mark*.rpm'
+def runtests(String javaVersion){
   sh label:'test', script: '''#!/bin/bash
     export JAVA_HOME=$'''+javaVersion+'''
     export GRADLE_USER_HOME=$WORKSPACE/$GRADLE_DIR
     export PATH=$GRADLE_USER_HOME:$JAVA_HOME/bin:$PATH
     cd marklogic-spark-connector
     echo "mlPassword=admin" > gradle-local.properties
+    sleep 30s
```

Comment:
Why is this here? If it's needed, add a comment before it to explain why it's sleeping and why 30s is the chosen value.

Reply:
Added a comment.